### ISSUE TITLE
ENOENT: no such file or directory

### ISSUE THREAD

**Issue by RafidMuhymin** [ACP-CODE/astro-favicons#4]
I am getting the following error when running `astro build`. `astro dev` works fine.

```bash
[Error: ENOENT: no such file or directory, mkdir 'E:\E:\windows\dilmahtea.me\public'] {
  errno: -4058,
  code: 'ENOENT',
  syscall: 'mkdir',
  path: 'E:\\E:\\windows\\dilmahtea.me\\public'
}
```

Configuration:

```js
export default defineConfig({
  integrations: [
    favicons({
      masterPicture: "./src/favicon.svg",
      emitAssets: true,
      faviconsDarkMode: true,
    }),
  ],
});
```

**Comment by JakiChen** [comment]
> I am getting the following error when running `astro build`. `astro dev` works fine.
> 
> ```shell
> [Error: ENOENT: no such file or directory, mkdir 'E:\E:\windows\dilmahtea.me\public'] {
>   errno: -4058,
>   code: 'ENOENT',
>   syscall: 'mkdir',
>   path: 'E:\\E:\\windows\\dilmahtea.me\\public'
> }
> ```
> 
> Configuration:
> 
> ```js
> export default defineConfig({
>   integrations: [
>     favicons({
>       masterPicture: "./src/favicon.svg",
>       emitAssets: true,
>       faviconsDarkMode: true,
>     }),
>   ],
> });
> ```

**Thank you for your feedback, this may be fixed in [astro-favicons^1.1.1](https://github.com/ACP-CODE/astro-favicons/blob/91503061ca9efe65ad8e0161e1f045a46d14e770/src/core.ts#L100)**

**Comment by JakiChen** [comment]
> I am getting the following error when running `astro build`. `astro dev` works fine.
> 
> ```shell
> [Error: ENOENT: no such file or directory, mkdir 'E:\E:\windows\dilmahtea.me\public'] {
>   errno: -4058,
>   code: 'ENOENT',
>   syscall: 'mkdir',
>   path: 'E:\\E:\\windows\\dilmahtea.me\\public'
> }
> ```
> 
> Configuration:
> 
> ```js
> export default defineConfig({
>   integrations: [
>     favicons({
>       masterPicture: "./src/favicon.svg",
>       emitAssets: true,
>       faviconsDarkMode: true,
>     }),
>   ],
> });
> ```

**Thank you for your feedback, this may be fixed in [astro-favicons^1.1.1](https://github.com/ACP-CODE/astro-favicons/blob/91503061ca9efe65ad8e0161e1f045a46d14e770/src/core.ts#L100)**
